.github/workflows/release.yml

```diff
@@ -56,7 +60,7 @@ jobs:
           install_url: https://releases.nixos.org/nix/nix-2.13.3/install
           extra_nix_config: |
             access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
-            substituters = http://cache.nixos.org https://hydra.iohk.io
+            substituters = http://cache.nixos.org https://cache.iog.io
```

Comment:
Worth asking the question of what should be standard here K uses the following. https://github.com/runtimeverification/k/blob/develop/.github/workflows/release.yml#L37 
A couple weeks ago anywhere I found the nix install I tried to update these to use the same references. 
Is cache.iog.io faster? Have something haskell needs? Will this be an issue in 'k' if there is a discrepancy building K?

Reply:
Good question - maybe @goodlyrottenapple can confirm .
The trusted key is for `hydra.iohk.io` but AFAIK this `iog.io` is the newer name for the same thing.

Reply:
From googling around, it seems `cache.iog.io` is indeed the better choice - [the manual](https://input-output-hk.github.io/haskell.nix/tutorials/getting-started.html) got [updated about 11 months ago](https://github.com/input-output-hk/haskell.nix/commit/b1661f08e3d6337b11bf7152bc02e314a76cc557).